src/pages/article/index.tsx

```diff
@@ -1,19 +1,47 @@
-import { ParsedUrlQuery } from 'querystring';
 import type { GetServerSideProps, InferGetServerSidePropsType } from 'next';
+import {
+  dehydrate,
+  DehydratedState,
+  HydrationBoundary,
+  QueryClient,
+} from '@tanstack/react-query';
 import { Article } from '@/components/article/Article';
+import { QueryKeys } from '@/api/news/queries';
+import { fetchArticleByUri } from '@/api/news/module';
 
-export const getServerSideProps = (async ({ query }) => ({
-  props: {
-    uri: query.uri,
-  },
-})) satisfies GetServerSideProps<{ uri: ParsedUrlQuery[string] }>;
+export const getServerSideProps = (async ({ query }) => {
+  const { uri } = query;
 
-export default function ArticlePage({
-  uri,
-}: InferGetServerSidePropsType<typeof getServerSideProps>) {
   if (!(uri && typeof uri === 'string')) {
-    return <div>Article not found</div>;
+    return {
+      notFound: true,
+    };
   }
 
-  return <Article uri={uri}></Article>;
+  const queryClient = new QueryClient();
```

Comment:
move queryClient to `src/constants/query-client.ts`

Reply:
done